intro/integration/js.md

```diff
@@ -64,3 +70,144 @@ If you are using `ionic start` to create an Ionic project, you have to customize
 ## Angular
 
 If you are using `ng new` to bootstrap your app, you will find it uses Webpack internally to build the project. Unfortunately, there is no way to customize the build process and therefore not possible to make it work with Skygear. We recommend using [Angular-webpack](https://github.com/preboot/angular-webpack) for bootstrapping, and then follow the above guide for integrating with Webpack.
```

Comment:
I think he means the Angular-Webpack guide. Changing "above" to "its" should fix it.